services/core-web/src/components/modalContent/ViewIncidentModal.js

```diff
@@ -38,16 +38,20 @@ export class ViewIncidentModal extends Component {
     const formattedPhoneNo = this.props.incident.reported_by_phone_ext
       ? `${this.props.incident.reported_by_phone_no} ext: ${this.props.incident.reported_by_phone_ext}`
       : this.props.incident.reported_by_phone_no;
+
     return (
       <div>
         <h5>Initial Report</h5>
         <div className="content--light-grey padding-small">
           <div className="inline-flex padding-small">
-            <p className="field-title">Incident type</p>
+            <p className="field-title">Incident type(s)</p>
             <p>
-              {this.props.incidentCategoryCodeHash[
-                this.props.incident.mine_incident_category_code
-              ] || Strings.EMPTY_FIELD}
+              {this.props.incident.categories && this.props.incident.categories.length > 0
+                ? this.props.incident.categories
+                    .sort((a, b) => (a.display_order > b.display_order ? 1 : -1))
+                    .map((c) => this.props.incidentCategoryCodeHash[c.mine_incident_category_code])
+                    .join(", ")
+                : Strings.EMPTY_FIELD}
```

Comment:
Display the incident categories sorted in ascending order by their display_order attribute.